ntoskrnl/ke/i386/kiinit.c

```diff
@@ -432,31 +485,28 @@ KiInitializeKernel(IN PKPROCESS InitProcess,
     {
         /* Set it always on */
         SharedUserData->NXSupportPolicy = NX_SUPPORT_POLICY_ALWAYSON;
-        FeatureBits |= KF_NX_ENABLED;
+        KeFeatureBits |= KF_NX_ENABLED;
```

Comment:
Now you are modifying the global `KeFeatureBits` with state from every processor, instead of it being the bits of the boot CPU only. Not sure this is what you want, this changes what this global variable was designed for.

Reply:
Yeah, I'm aware of that. I left it like that, because I'm going to do more refactorings of Ke initialization code in preparation for SMP. Want to unify Ke stuff across architectures and make it less messy

Reply:
I think each commit/PR should strive to give a sane state afterwards already, if possible.